gemp-lotr/gemp-lotr-cards/src/main/resources/cards/unofficial/pc/setV02/setv02-Gandalf-playtest.hjson

```diff
@@ -119,23 +119,51 @@
 		vitality: 4
 		signet: Theoden
 		resistance: 6
-		/*requires: {
-			
-		}
-		effects: [
+		effects: {
 			{
-				
+				type: modifier
+				modifier: {
+					type: modifystrength
+					filter: name(Theoden)
+					amount: 1
+				}
 			}
 			{
-				
+				type: modifier
+				modifier: {
+					type: AddKeyword
+				    filter: mounted
+					keyword: valiant
+
+				}
 			}
-		]*/
-		gametext: Theoden is strength +1.<br>Each mounted companion gains <b>valiant</b>.<br>Response: If a minion's special ability is used (except during a skirmish), spot 3 valiant companions and exert Gandalf to prevent that and wound that minion. 
+			{
+				type: activatedTrigger
+				trigger: {
+					type: UsesSpecialAbility
+					memorize: actingminion
+					filter: not(skirmish)
```

Comment:
This should be replace with:
filter: minion
requires: [
  {
     type: not
     requires: {
        type: phase
        phase: skirmish
     }
  }
  {
     type: canSpot
     filter: valiant,companion
     count: 3
  }
]